src/plugins/viewport.ts

```diff
@@ -49,6 +49,18 @@ export default class Viewport implements IPlugin {
     return viewport;
   }
 
+  private getDocumentHeight(body, documentElement): number {
```

Comment:
>getDocumentHeight [](start = 10, length = 17)

Please add a short explanation why this extra complexity is needed vs. what we had before